flytekit/configuration/sdk.py

```diff
@@ -32,47 +18,6 @@
 clean up data in these directories.
 """
 
-SDK_PYTHON_VENV = _config_common.FlyteStringListConfigurationEntry("sdk", "python_venv", default=[])
-"""
-This is a list of commands/args which will be prefixed to the entrypoint command by SDK.
-"""
-
-ROLE = _config_common.FlyteStringConfigurationEntry("sdk", "role")
-"""
-This is the role the SDK will use by default to execute workflows.  For example, in AWS this should be an IAM role
-string.
-"""
-
-NAME_FORMAT = _config_common.FlyteStringConfigurationEntry("sdk", "name_format", default="{module}.{name}")
-"""
-This is a Python format string which the SDK will use to generate names for discovered entities.  The default is
-'{module}.{name}' which will result in strings like 'package.module.name'.  Any template portion of the string can only
-include 'module' or 'name'.  So '{name}' is valid, but '{key}' is not.
-"""
-
-TASK_NAME_FORMAT = _config_common.FlyteStringConfigurationEntry("sdk", "task_name_format", fallback=NAME_FORMAT)
-"""
-This is a Python format string which the SDK will use to generate names for tasks. Any template portion of the
-string can only include 'module' or 'name'.  So '{name}' is valid, but '{key}' is not. If not specified,
-we fall back to the configuration for :py:attr:`flytekit.configuration.sdk.NAME_FORMAT`
-"""
-
-WORKFLOW_NAME_FORMAT = _config_common.FlyteStringConfigurationEntry("sdk", "workflow_name_format", fallback=NAME_FORMAT)
-"""
-This is a Python format string which the SDK will use to generate names for workflows. Any template portion of the
-string can only include 'module' or 'name'.  So '{name}' is valid, but '{key}' is not. If not specified,
-we fall back to the configuration for :py:attr:`flytekit.configuration.sdk.NAME_FORMAT`
-"""
-
-LAUNCH_PLAN_NAME_FORMAT = _config_common.FlyteStringConfigurationEntry(
-    "sdk", "launch_plan_name_format", fallback=NAME_FORMAT
-)
-"""
-This is a Python format string which the SDK will use to generate names for launch plans. Any template portion of the
```

Comment:
are these leftovers from the legacy code removal?

Reply:
yup